Aggiunto capitolo CV per un developer (#112)
Il-Libro-Open-Source/book: docs/it/linkedin.md

**Il profilo LinkedIn**

Assieme al CV, anche **LinkedIn** è uno strumento estremamente potente per cercare lavoro, costruire network, creare relazioni.
L'obiettivo è quello di fornire degli spunti utili per rendere il tuo profilo LinkedIn attrattivo ma soprattutto "keyword-friendly". Se non sei in cerca di lavoro, questo capitolo può comunque esserti utile per meglio posizionare il tuo profilo, renderlo più professionale e può fornirti qualche spunto e curiosità.

**1) Indicazioni generali**

> LinkedIn, nel mondo, conta circa 830 milioni di utenti attivi, di cui circa 16 milioni in Italia. Per il settore tech è, senza dubbio, lo strumento migliore che le persone che lavorano come recruiter/head hunters/talent acquisition usano tutti i giorni per trovare professionisti/e del settore IT. Prima di scendere nel dettaglio dei vari punti della guida, alcune considerazioni generali per iniziare al meglio:

Reviewer: > LinkedIn, nel mondo, conta circa 830 milioni di utenti attivi, di cui circa 16 milioni in Italia.

Fonte?

Author: inserita fonte

> -Moltissimi recruiter non conoscono il gergo tech, i nomi delle tecnologie/framework, confondono Java con JavaScript (si, succede ancora) e così via. E questo, immagino, potrebbe non farti piacere. Ma cerca comunque di essere sempre comprensivo ed empatico verso chi ti scrive. Cerca, per quanto possibile, di rispondere sempre ai messaggi che ricevi perché dall'altra parte c'è comunque una persona che sta lavorando e che merita rispetto esattamente come lo meriti tu. Se al momento non sei in cerca di un nuovo lavoro, ti consiglio magari di prepararti un testo preimpostato da inviare a chi ti scrive, da usare banalmente come risposta. Tu perdi pochi secondi di tempo ma ti assicuro che chi c'è dall'altra parte apprezzerà!

Reviewer: ```suggestion
- Moltissimi recruiter non conoscono il gergo tech, i nomi delle tecnologie/framework, confondono Java con JavaScript (si, succede ancora) e così via. E questo, immagino, potrebbe non farti piacere. Ma cerca comunque di essere sempre comprensivo ed empatico verso chi ti scrive. Cerca, per quanto possibile, di rispondere sempre ai messaggi che ricevi perché dall'altra parte c'è comunque una persona che sta lavorando e che merita rispetto esattamente come lo meriti tu. Se al momento non sei in cerca di un nuovo lavoro, ti consiglio magari di prepararti un testo preimpostato da inviare a chi ti scrive, da usare banalmente come risposta. Tu perdi pochi secondi di tempo ma ti assicuro che chi c'è dall'altra parte apprezzerà!
```

2. **Foto profilo e foto copertina**
   Rispetto al CV, su LinkedIn la foto profilo è FONDAMENTALE (uno studio rivela che chi ha una foto profilo professionale ha 20x visualizzazioni del profilo rispetto a chi non ne ha). Usane una professionale (NO foto sgranate, NO fototessere di 20 anni fa, NO foto al mare, etc..). Come avrai potuto leggere nel capitolo dedicato al CV, se non hai una foto anche un avatar o una illustrazione vanno bene. Riguardo l'immagine di copertina: non è fondamentale ma la si può sicuramente per dare un tocco di personalizzazione al tuo profilo: un pezzettino di codice? Una quote famosa? Un meme? Ecco, puoi usare l'immagine di copertina per "trasmettere qualcosa" a chi visita il tuo profilo. Sfruttala!

**3) La headline**
Immediatamente dopo aver settato la foto profilo e l'immagine di copertina, bisogna porre la massima attenzione a quest'altra importantissima sezione, ovvero la headline: è la sezione testuale che si trova immediatamente sotto la foto, che di norma comprende il sommario e la posizione lavorativa attuale.

> Come è composta una headline che funziona? Semplice: fornisce in maniera veloce ma efficace una overview prima di tutto dello stack tecnologico che utilizza e conosce la persona, ma anche un ulteriore spunto riguardo eventuali passioni (es: per l'open source).

Reviewer: ```suggestion
Come è composta una headline che funziona? Semplice: fornisce in maniera veloce ma efficace una panoramica prima di tutto dello stack tecnologico che utilizza e conosce la persona, ma anche un ulteriore spunto riguardo eventuali passioni (ad esempio: per l'open source).
```

- Content is king: sarò ridondante ma più informazioni scrivi sul profilo meglio é: meglio un profilo ben curato, con tante informazioni, che uno scarno e vuoto. Ti assicuro che l'effort sulla compilazione del proprio profilo viene notato!

**6) La formazione**
La sezione del profilo LinkedIn dedicata alla formazione viene spesso tralasciata, ma in realtà è importante tanto quanto le altre sezioni. È una delle sezioni, infatti, che viene "letta" dai vari filtri di Linkedin Recruiter nel momento in cui i recruiter impostano una ricerca mirata (ad esempio: uno specifico titolo di studio, una particolare università o uno specifico indirizzo). Quindi presta molta attenzione a questa sezione e compilala in maniera esaustiva. Inserisci quindi il nome della facoltà, l'indirizzo scelto, il voto di laurea, ed i nomi degli esami che a tuo avviso potrebbero essere affini al tuo percorso professionale.

> Se stai frequentando una Academy/Bootcamp IT, ovviamente inserirlo ti farà apparire come studente di quella academy, e sicuramente aiuta nell'indicizzazione del tuo profilo (anche perché, scrivendo le parole chiave relative alle tecnologie/linguaggi che stai studiando, inserisci ulteriori parole chiave sul tuo profilo che, come avrai capito, sono fondamentali).

Reviewer: ```suggestion
Se stai frequentando una Academy o un Bootcamp IT, ovviamente inserirlo ti farà apparire come studente di quella academy, e sicuramente aiuta nell'indicizzazione del tuo profilo (anche perché, scrivendo le parole chiave relative alle tecnologie/linguaggi che stai studiando, inserisci ulteriori parole chiave sul tuo profilo che, come avrai capito, sono fondamentali).
```

**7) Licenze e certificazioni**

> Questa sezione aiuta a mostrare il percorso formativo in ambito di corsi/certificazioni/attestati formativi. Aver preso determinate certificazioni è visto sicuramente come un grosso plus da alcune aziende, soprattutto se queste certificazioni sono "prestigiose".Averle sul profilo è quindi sicuramente un plus. Alcune aziende richiedono obbligatoriamente delle certificazioni (ad esempio la Green Belt Six Sigma, ormai imprescindibile per le basi di un buon Project Manager, o le certificazioni cloud AWS/Azure/Google Cloud) per accedere alle loro selezioni o per un avanzamento di carriera.

Reviewer: ```suggestion
Questa sezione aiuta a mostrare il percorso formativo in ambito di corsi/certificazioni/attestati formativi. Aver preso determinate certificazioni è visto sicuramente come un grosso plus da alcune aziende, soprattutto se queste certificazioni sono "prestigiose". Averle sul profilo è quindi sicuramente un valore aggiunto. Alcune aziende richiedono obbligatoriamente delle certificazioni (ad esempio la Green Belt Six Sigma, ormai imprescindibile per le basi di un buon Project Manager, o le certificazioni cloud AWS/Azure/Google Cloud) per accedere alle loro selezioni o per un avanzamento di carriera.
```

**8) Competenze**
Molto spesso sui profili LinkedIn viene notata dai recruiter la scarsa cura di questa sezione, che in realtà è estremamente importante, assieme alla sezione sulle esperienze professionali. Qual è la parola che ci sta seguendo in tutto questo capitolo? KEYWORD. Ecco, è esattamente la sezione competenze che alimenta in maniera decisamente importante il vostro comparire (o meno) fra le ricerche dei recruiter.

La sezione competenze DEVE essere curata e tenuta in ordine per vari motivi:

- Aggiorna costantemente la sezione, eliminando tecnologie/linguaggi/framework che non usi più (o che conosci ma non vuoi più utilizzare professionalmente) per evitare di ricevere proposte di lavoro con tecnologie che non ti interessano.
- Una cosa importantissima è ordinare le competenze: metti in cima le competenze sulle quali sei più forte e alla fine quelle dove ti senti meno competente.

> - LinkedIn ti dà la possibilità di mettere "in evidenza" la tua top three di competenze, che appariranno subito sul tuo profilo: ricollegandomi quindi al punto precedente, scegli le tre competenze dove ti senti più skillato e spostale in alto.

Reviewer: ```suggestion
- LinkedIn ti dà la possibilità di mettere "in evidenza" la tua **Top 3 di competenze**, che appariranno subito sul tuo profilo: ricollegandomi quindi al punto precedente, scegli le tre competenze dove ti senti più esperto e spostale in alto.
```

**Il profilo LinkedIn**

Assieme al CV, anche **LinkedIn** è uno strumento estremamente potente per cercare lavoro, costruire network, creare relazioni.
L'obiettivo è quello di fornire degli spunti utili per rendere il tuo profilo LinkedIn attrattivo ma soprattutto "keyword-friendly". Se non sei in cerca di lavoro, questo capitolo può comunque esserti utile per meglio posizionare il tuo profilo, renderlo più professionale e può fornirti qualche spunto e curiosità.

**1) Indicazioni generali**
LinkedIn, nel mondo, conta circa 830 milioni di utenti attivi, di cui circa 17 milioni in Italia (fonte: https://www.linkedin.com/pulse/utenti-linkedin-italia-e-nel-mondo-/?originalSubdomain=it).

> Per il settore tech è, senza dubbio, lo strumento migliore che le persone che lavorano come recruiter/head hunters/talent acquisition usano tutti i giorni per trovare professionisti/e del settore IT. Prima di scendere nel dettaglio dei vari punti della guida, alcune considerazioni generali per iniziare al meglio:

Reviewer 2: ```suggestion
Per il settore tech è, senza dubbio, lo strumento migliore che le persone che lavorano come recruiter/head hunter/talent acquisition usano tutti i giorni per trovare professionisti/e del settore IT. Prima di scendere nel dettaglio dei vari punti della guida, alcune considerazioni generali per iniziare al meglio:
```

> - Moltissimi recruiter non conoscono il gergo tech, i nomi delle tecnologie/framework, confondono Java con JavaScript (si, succede ancora) e così via. E questo, immagino, potrebbe non farti piacere. Ma cerca comunque di essere sempre comprensivo ed empatico verso chi ti scrive. Cerca, per quanto possibile, di rispondere sempre ai messaggi che ricevi perché dall'altra parte c'è comunque una persona che sta lavorando e che merita rispetto esattamente come lo meriti tu. Se al momento non sei in cerca di un nuovo lavoro, ti consiglio magari di prepararti un testo preimpostato da inviare a chi ti scrive, da usare banalmente come risposta. Tu perdi pochi secondi di tempo ma ti assicuro che chi c'è dall'altra parte apprezzerà!

Reviewer 3: ```suggestion
- Moltissimi recruiter non conoscono il gergo tech, i nomi delle tecnologie/framework, confondono Java con JavaScript (sì, succede ancora) e così via. E questo, immagino, potrebbe non farti piacere. Ma cerca comunque di avere comprensione ed empatia verso chi ti scrive. Cerca, per quanto possibile, di rispondere sempre ai messaggi che ricevi perché dall'altra parte c'è comunque una persona che sta lavorando e che merita rispetto esattamente come lo meriti tu. Se al momento non sei in cerca di un nuovo lavoro, ti consiglio magari di prepararti un testo preimpostato da inviare a chi ti scrive, da usare banalmente come risposta. Tu perdi pochi secondi di tempo ma ti assicuro che chi c'è dall'altra parte apprezzerà!
```

> - La questione dell'opentowork: sicuramente penserai che la cornice verde #opentowork non sia un granché, vero? E pensi anche che, se la attivi, la tua azienda ti sgamerà subito e capirà che stai cercando un nuovo lavoro. Non so se lo sai, ma c'è anche la possibilità di segnalare che sei opentowork ma non al "pubblico": basta cliccare sul tasto "disponibile per" sotto la foto profilo e, dopo aver selezionato l'opzione "cambiare lavoro" nella maschera che ti si aprirà, scorri in basso e troverai un menù con il titolo "scegli chi può vedere che sei disponibile per opportunità di lavoro" e seleziona "solo recruiter": in questo modo i recruiter, utilizzando Linkedin Recruiter, vedranno una voce nel tuo profilo "disponibile a lavorare".

Reviewer 2: Siamo sicuri di voler aggiungere gli step per impostare questa voce? Si corre il rischio di dover sottoporre il testo a continue revisioni se LinkedIn decidesse di cambiare il path. Io mi limiterei al generale

Author: Dubito che LinkedIn modificherà quel path onestamente, è una funzione che esiste da tanto e che è parte fondamentale della ricerca tramite LinkedIn recruiter :)

Reviewer 4: Beh, a sto punto indicherei solo con una nota che queste istruzioni sono valide alla stesura del documento e che potrebbero aver subito variazioni.

> - Chi cerca professionisti su Linkedin lo fa principalmente utilizzando le keyword (parole chiave), in due modi: o usando le stringhe booleane o usando appunto le parole chiave ed altri filtri con la licenza LinkedIn Recruiter (il profilo a pagamento di Linkedin che sblocca tutta una serie di funzionalità, fra cui tantissimi filtri di ricerca ottimizzata e molto altro). Da qui si intuisce che TUTTO ciò che è scritto sul tuo profilo è utile a farti trovare e, soprattutto, a rientrare in modo mirato nelle ricerche che vengono effettuate sulla piattaforma proprio tramite le parole chiave. Approfondiremo comunque questo aspetto nelle sezioni successive della guida.

Reviewer 2: Cosa sono le stringhe booleane? Forse converrebbe spiegare?

Author: Vogliamo integrare con questo link?
https://www.linkedin.com/help/linkedin/answer/a524335

Reviewer 2: Ah ora ho capito! Si dai ci sta per evitare magari di entrare troppo in tecnicismi

> - Chi cerca professionisti su Linkedin lo fa principalmente utilizzando le keyword (parole chiave), in due modi: o usando le [stringhe booleane](https://www.linkedin.com/help/linkedin/answer/a524335) o usando appunto le parole chiave ed altri filtri con la licenza LinkedIn Recruiter (il profilo a pagamento di Linkedin che sblocca tutta una serie di funzionalità, fra cui tantissimi filtri di ricerca ottimizzata e molto altro). Da qui si intuisce che TUTTO ciò che è scritto sul tuo profilo è utile a farti trovare e, soprattutto, a rientrare in modo mirato nelle ricerche che vengono effettuate sulla piattaforma proprio tramite le parole chiave. Approfondiremo comunque questo aspetto nelle sezioni successive della guida.

Reviewer 5: ```suggestion
- Chi cerca professionisti/e su Linkedin lo fa principalmente utilizzando le keyword (parole chiave), in due modi: o usando le [stringhe booleane](https://www.linkedin.com/help/linkedin/answer/a524335) o usando appunto le parole chiave ed altri filtri con la licenza LinkedIn Recruiter (il profilo a pagamento di Linkedin che sblocca tutta una serie di funzionalità, fra cui tantissimi filtri di ricerca ottimizzata e molto altro). Da qui si intuisce che TUTTO ciò che è scritto sul tuo profilo è utile a farti trovare e, soprattutto, a rientrare in modo mirato nelle ricerche che vengono effettuate sulla piattaforma proprio tramite le parole chiave. Approfondiremo comunque questo aspetto nelle sezioni successive della guida.
```

> - LinkedIn, come ogni altro social network, ha un proprio algoritmo che regola cosa ti appare nel feed, le interazioni fra iscritti e molto altro. Per mantenere un profilo visibile ed "engaging" molte guide online consigliano di postare 3-4 contenuti a settimana e di lasciare almeno due commenti al giorno. Questa, almeno, sarebbe la golden rule per tenere un profilo "amico" dell'algoritmo. Detto questo, sei libero di utilizzare il tuo profilo come meglio credi: se non hai nulla di interessante da postare, non devi farlo per forza! Il grandissimo pregio di LinkedIn è dato proprio dai contenuti di valore che vengono postati. Trova quindi il tuo personale "tone of voice" ed adattalo a quello che vuoi comunicare. E ricorda: anche non comunicare è comunicare!

Reviewer 3: ```suggestion
- LinkedIn, come ogni altro social network, ha un proprio algoritmo che regola cosa ti appare nel feed, le interazioni fra iscritti e molto altro. Per mantenere un profilo visibile ed "engaging" molte guide online consigliano di postare 3-4 contenuti a settimana e di lasciare almeno due commenti al giorno. Questa, almeno, sarebbe la golden rule per tenere un profilo "amico" dell'algoritmo. Detto questo, hai tutta la libertà di utilizzare il tuo profilo come meglio credi: se non hai nulla di interessante da postare, non devi farlo per forza! Il grandissimo pregio di LinkedIn è dato proprio dai contenuti di valore che vengono postati. Trova quindi il tuo personale "tone of voice" ed adattalo a quello che vuoi comunicare. E ricorda: anche non comunicare è comunicare!
```

2. **Foto profilo e foto copertina**

>    Rispetto al CV, su LinkedIn la foto profilo è FONDAMENTALE (uno studio rivela che chi ha una foto profilo professionale ha 20x visualizzazioni del profilo rispetto a chi non ne ha). Usane una professionale (NO foto sgranate, NO fototessere di 20 anni fa, NO foto al mare, etc..). Come avrai potuto leggere nel capitolo dedicato al CV, se non hai una foto anche un avatar o una illustrazione vanno bene. Riguardo l'immagine di copertina: non è fondamentale ma la si può sicuramente per dare un tocco di personalizzazione al tuo profilo: un pezzettino di codice? Una quote famosa? Un meme? Ecco, puoi usare l'immagine di copertina per "trasmettere qualcosa" a chi visita il tuo profilo. Sfruttala!

Reviewer 3: ```suggestion
   Rispetto al CV, su LinkedIn la foto profilo è FONDAMENTALE (uno studio rivela che chi ha una foto profilo professionale ha 20x visualizzazioni del profilo rispetto a chi non ne ha). Usane una professionale (NO foto sgranate, NO fototessere di 20 anni fa, NO foto al mare, etc..). Come avrai potuto leggere nel capitolo dedicato al CV, se non hai una foto anche un avatar o una illustrazione vanno bene. Riguardo l'immagine di copertina: non è fondamentale ma la si può sicuramente sfruttare per dare un tocco di personalizzazione al tuo profilo: un pezzettino di codice? Una quote famosa? Un meme? Ecco, puoi usare l'immagine di copertina per "trasmettere qualcosa" a chi visita il tuo profilo. Sfruttala!
```

>    Rispetto al CV, su LinkedIn la foto profilo è FONDAMENTALE (uno studio rivela che chi ha una foto profilo professionale ha 20x visualizzazioni del profilo rispetto a chi non ne ha). Usane una professionale (NO foto sgranate, NO fototessere di 20 anni fa, NO foto al mare, etc..). Come avrai potuto leggere nel capitolo dedicato al CV, se non hai una foto anche un avatar o una illustrazione vanno bene. Riguardo l'immagine di copertina: non è fondamentale ma la si può sicuramente sfruttare per dare un tocco di personalizzazione al tuo profilo: un pezzettino di codice? Una quote famosa? Un meme? Ecco, puoi usare l'immagine di copertina per "trasmettere qualcosa" a chi visita il tuo profilo. Sfruttala!

Reviewer 5: ```suggestion
   Rispetto al CV, su LinkedIn la foto profilo è FONDAMENTALE (uno studio rivela che chi ha una foto profilo professionale ha 20x visualizzazioni del profilo rispetto a chi non ne ha). Usane una professionale (NO foto sgranate, NO fototessere di 20 anni fa, NO foto al mare, ecc..). Come avrai potuto leggere nel capitolo dedicato al CV, se non hai una foto anche un avatar o una illustrazione vanno bene. Riguardo l'immagine di copertina: non è fondamentale ma la si può sicuramente sfruttare per dare un tocco di personalizzazione al tuo profilo: un pezzettino di codice? Una quote famosa? Un meme? Ecco, puoi usare l'immagine di copertina per "trasmettere qualcosa" a chi visita il tuo profilo. Sfruttala!
```

Inoltre, e questa è una cosa importantissima, tutte le tecnologie/framework inserite nella headline sono delle keyword (ne abbiamo parlato nell'introduzione)
A prescindere dalle keyword, la headline è a colpo d'occhio una delle primissime cose che si notano sul profilo: è uno spazio dove devi riuscire a comunicare in poche righe chi sei, cosa fai (ed eventualmente, cosa cerchi). Va quindi sfruttata AL MASSIMO!

**4) Sezione informazioni**
Esattamente come l'about me sul curriculum, questa sezione sul tuo profilo è utilissima per fornire una overview approfondita delle tue esperienze lavorative, i linguaggi/tecnologie/framework che conosci e che utilizzi, obiettivi futuri, attività che segui giornalmente e perché no, anche informazioni che potrebbero sembrare poco utili (hobbies, cosa ti piace fare nel tempo libero, i libri che stai leggendo, le persone che segui, i progetti che sviluppi nel tempo libero) ma che forniscono a chi legge una vista più completa e dettagliata su di te.

> La sezione informazioni dovrebbe contiene tutte le informazioni che servono ad un/una recruiter: una breve descrizione di sè e dell'approccio al lavoro, un elenco esaustivo di tecnologie/linguaggi/framework/software/metodologie ed anche ulteriori input riguardo la gestione del team, la capacità di seguire colloqui tecnici e l'abitudine al lavoro Agile.

Reviewer 3: ```suggestion
La sezione informazioni dovrebbe contenere tutte le informazioni che servono ad un/una recruiter: una breve descrizione di sè e dell'approccio al lavoro, un elenco esaustivo di tecnologie/linguaggi/framework/software/metodologie ed anche ulteriori input riguardo la gestione del team, la capacità di seguire colloqui tecnici e l'abitudine al lavoro Agile.
```

> La sezione informazioni dovrebbe contenere tutte le informazioni che servono ad un/una recruiter: una breve descrizione di sè e dell'approccio al lavoro, un elenco esaustivo di tecnologie/linguaggi/framework/software/metodologie ed anche ulteriori input riguardo la gestione del team, la capacità di seguire colloqui tecnici e l'abitudine al lavoro Agile.

Reviewer 5: ```suggestion
La sezione "informazioni" dovrebbe contenere tutte le informazioni che servono ad un/una recruiter: una breve descrizione di sè e dell'approccio al lavoro, un elenco esaustivo di tecnologie/linguaggi/framework/software/metodologie ed anche ulteriori input riguardo la gestione del team, la capacità di seguire colloqui tecnici e l'abitudine al lavoro Agile.
```

- Aggancia alla posizione lavorativa le competenze: lo vedremo nella sezione dedicata più avanti, ma scrivere le competenze sul profilo è FONDAMENTALE per farsi trovare dai recruiter. Agganciare quindi le competenze al lavoro che si sta svolgendo (o si è svolto in precedenza) è utilissimo per dimostrare uno "storico" professionale coerente con quello che fai attualmente o con quello che hai fatto in precedenza.
- Cerca di "agganciare" correttamente il nome dell'azienda per la quale lavori. Spesso trovo profili con palesi errori di battitura riguardo l'azienda per la quale si lavora, di conseguenza LinkedIn non mostra il logo corretto dell'azienda. Ne consegue che un recruiter che, ad esempio, voglia andare a leggere di cosa si occupa l'azienda, il settore, le dimensioni ed altro, banalmente non potrà farlo. Poni quindi attenzione a selezionare correttamente l'azienda per la quale lavori, è importantissimo!

> - Content is king: sarò ridondante ma più informazioni scrivi sul profilo meglio é: meglio un profilo ben curato, con tante informazioni, che uno scarno e vuoto. Ti assicuro che l'effort sulla compilazione del proprio profilo viene notato!

Reviewer 5: ```suggestion
- **Content is king**: sarò ridondante, ma più informazioni scrivi sul profilo meglio è: meglio un profilo ben curato, con tante informazioni, che uno scarno e vuoto. Ti assicuro che l'effort sulla compilazione del proprio profilo viene notato!
```

- Aggiorna costantemente la sezione, eliminando tecnologie/linguaggi/framework che non usi più (o che conosci ma non vuoi più utilizzare professionalmente) per evitare di ricevere proposte di lavoro con tecnologie che non ti interessano.
- Una cosa importantissima è ordinare le competenze: metti in cima le competenze sulle quali sei più forte e alla fine quelle dove ti senti meno competente.
- LinkedIn ti dà la possibilità di mettere "in evidenza" la tua **Top 3 di competenze**, che appariranno subito sul tuo profilo: ricollegandomi quindi al punto precedente, scegli le tre competenze dove ti senti più esperto e spostale in alto.
- Avevamo parlato delle competenze anche nel punto 5, le esperienze professionali: LinkedIn ti permette di "legare" le competenze all'esperienza professionale, super utile per dimostrare che competenze stai utilizzando nell'attuale ruolo che ricopri.
- Chiedi a colleghi/amici/collegamenti di confermarti le competenze: è ovviamente una pura vanity metric ma probabilmente sai meglio di me che "l'effetto wow" di vedere una competenza confermata da tante persone funziona.

**9) Lingue**

> Esattamente come per il CV, la sezione lingue fornisce la possibilità di comunicare la tua conoscenza di una o più lingue che non siano la madrelingua (nel nostro caso, ovviamente, l'italiano). Mi capita ad esempio di gestire selezioni dove è assolutamente obbligatoria la conoscenza dell'inglese (perché magari il team è internazionale, l'azienda ha clienti esteri, si sta espandendo fuori dall'Italia, etc.), di conseguenza dimostrare una conoscenza di una lingua straniera mi aiuta a capire se il profilo che sto visitando soddisfa quel requisito. Se hai certificazioni o attestati di lingua, è assolutamente ottima cosa inserirli nella sezione di cui abbiamo parlato prima, ovvero "Licenze e certificazioni".

Reviewer 5: ```suggestion
Esattamente come per il CV, la sezione lingue fornisce la possibilità di comunicare la tua conoscenza di una o più lingue che non siano la madrelingua (nel nostro caso, ovviamente, l'italiano). Mi capita ad esempio di gestire selezioni dove è assolutamente obbligatoria la conoscenza dell'inglese (perché magari il team è internazionale, l'azienda ha clienti esteri, si sta espandendo fuori dall'Italia, ecc.), di conseguenza dimostrare una conoscenza di una lingua straniera mi aiuta a capire se il profilo che sto visitando soddisfa quel requisito. Se hai certificazioni o attestati di lingua, è assolutamente ottima cosa inserirli nella sezione di cui abbiamo parlato prima, ovvero "Licenze e certificazioni".
```

**10) Referenze**
Le recensioni sono un'arma potentissima che davvero in pochissimi sfruttano. È impossibile non avere un collega, manager, ex-capo, collaboratore, consulente esterno al quale poter chiedere una referenza. È davvero una richiesta che costa pochi minuti ed aiuta tantissimo ad arricchire il tuo profilo ed ad aumentare il tuo personal branding.

**11) Pubblicazioni**

> Come per il CV un'altra sezione di tutto rispetto sono le pubblicazioni. Questa sezione non solo aggiunge importanza al vostro profilo e lo arricchisce, ma può rilevarsi un elemento di rilievo in diversi casi. Può infatti essere particolarmente apprezzata da aziende orientate alla ricerca e all'innovazione. Le pubblicazioni possono quindi dimostrare la vostra propensione e volontà in questo ambito.
> Se avete pubblicazioni non esitate a pubblicarle!

Reviewer 3: ```suggestion
Come per il CV un'altra sezione di tutto rispetto sono le pubblicazioni. Questa sezione non solo aggiunge importanza al vostro profilo e lo arricchisce, ma può rilevarsi un elemento di rilievo in diversi casi. Può infatti essere particolarmente apprezzata da aziende orientate alla ricerca e all'innovazione. Le pubblicazioni possono quindi dimostrare la tua propensione e volontà in questo ambito.
Se hai pubblicazioni non esitare a pubblicarle!
```

Meglio usare la seconda persona singolare per coerenza col resto del capitolo

> Se hai pubblicazioni non esitare a pubblicarle!

Reviewer 5: ```suggestion
Se hai pubblicazioni non esitare a riportarle!
```
